.gitignore

```diff
@@ -1,3 +1,8 @@
 .idea
 ebs-terraform/.terraform/
 efs-terraform/.terraform/
+###script temp files
+log
+.cluster
+.env
+.yaml
```

Comment:
This seems a bit too broad. How about `/k8s/eks/*.yaml`? Or could we create that yaml in `.cluster` folder maybe?

Reply:
Yeah, we could tighten it down. I'll push the edit now.